src/DashboardService.Business/ChangeLog/GetAllChangeLogCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using LT.DigitalOffice.DashboardService.Business.ChangeLog.Interfaces;
+using LT.DigitalOffice.DashboardService.Data.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.ChangeLog.Filter;
+using LT.DigitalOffice.Kernel.Responses;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Business.ChangeLog;
+
+public class GetAllChangeLogCommand : IGetAllChangeLogsCommand
```

Comment:
Interface name IGetAllChangeLog**s**Command, but command name without it

Reply:
why? in example DepartmentService also

Reply:
Which command?

Reply:
Fix